test-utils/runtime-tester/src/scenario_builder.rs

```diff
@@ -0,0 +1,70 @@
+use crate::run_test::{BlockConfig, NetworkConfig, Scenario, TransactionConfig};
+use near_crypto::{InMemorySigner, KeyType};
+use near_primitives::{
+    transaction::Action,
+    types::{AccountId, BlockHeight, Nonce},
+};
+
+use std::str::FromStr;
+
+pub struct ScenarioBuilder {
+    height: BlockHeight,
+    nonce: Nonce,
+    scenario: Scenario,
+}
+
+impl ScenarioBuilder {
+    pub fn new(num_accounts: usize, use_in_memory_store: bool) -> Self {
+        let network_config =
+            NetworkConfig { seeds: (0..num_accounts).map(|x| id_to_seed(x)).collect() };
+
+        ScenarioBuilder {
+            height: 1,
+            nonce: 1,
```

Comment:
Should we start from 0 instead?

Reply:
No, I forgot why, but block at height 0 could not be produced